include/CLI/Option.hpp

```diff
@@ -682,41 +682,41 @@ class Option : public OptionBase<Option> {
     ///@{
 
     /// The number of arguments the option expects
-    int get_type_size() const { return type_size_min_; }
+    CLI11_NODISCARD int get_type_size() const { return type_size_min_; }
 
     /// The minimum number of arguments the option expects
-    int get_type_size_min() const { return type_size_min_; }
+    CLI11_NODISCARD int get_type_size_min() const { return type_size_min_; }
     /// The maximum number of arguments the option expects
-    int get_type_size_max() const { return type_size_max_; }
+    CLI11_NODISCARD int get_type_size_max() const { return type_size_max_; }
 
     /// Return the inject_separator flag
-    int get_inject_separator() const { return inject_separator_; }
```

Comment:
should the type of this be bool,  I noticed a few places this was used makes me think the error is in the return type here rather than in all the other spots?

Reply:
It does seem like it should be bool.